README.md

```diff
@@ -91,62 +18,50 @@ Watch logs for "sst" or "Quorum results", or run this quick check:
 for i in 0 1 2; do kubectl -n mysql exec mariadb-$i -- mysql -e "SHOW STATUS LIKE 'wsrep_cluster_size';" -N; done
 ```
 
-The value is also a metric in the [Prometheus](https://prometheus.io) endpoint:
+Port 9104 exposes plaintext metris in [Prometheus](https://prometheus.io/docs/concepts/data_model/) scrape format.
 ```
 # with kubectl -n mysql port-forward mariadb-0 9104:9104
 $ curl -s http://localhost:9104/metrics | grep ^mysql_global_status_wsrep_cluster_size
 mysql_global_status_wsrep_cluster_size 3
 ```
 
-### phpMyAdmin
-
-Carefully consider the security implications before you create this. Note that it uses a non-official image.
-
-```
-kubectl apply -f myadmin/
-```
-
-PhpMyAdmin has a login page where you need a mysql user. To allow login (with full access) create a user with your choice of password:
+A reasonable alert is on `mysql_global_status_wsrep_cluster_size` staying below the desired number of replicas.
 
-```
-kubectl -n mysql exec mariadb-0 -- mysql -e "CREATE USER 'phpmyadmin'@'%' IDENTIFIED BY 'my-admin-pw'; GRANT ALL ON *.* TO 'phpmyadmin'@'%';"
-```
+### Cluster un-health
 
-## Recover
+We need to assume a couple of things here. First and foremost:
+Production clusters are configured so that the statefulset pods do not go down together.
 
-It's in scope for this repo to support scaling down to 0 pods, then scale upp again (for example at maintenance or major upgrades). It's outside the scope of this repo to recover from crashes.
+ * Pods are properly spread across nodes.
+ * Nodes are spread across multiple availability zones.
 
-In the unlikely event that all pods crash, the statefulset tries to restart the pods but can fail with an infinite loop.
+Let's also assume that there is monitoring.
+Any `wsrep_cluster_size` issue (see above), or absence of `wsrep_cluster_size`
+should lead to a human being paged.
 
-In order to restart the cluster, the "Safe-To-Bootstrap" flag must be set. Please follow the instructions found [here](http://galeracluster.com/2016/11/introducing-the-safe-to-bootstrap-feature-in-galera-cluster/)
+Rarity combined with manual attention means that this statefulset can/should avoid
+attempts at automatic [recovery](http://galeracluster.com/documentation-webpages/pcrecovery.html).
+The reason for that being: we can't test for failure modes properly,
+as they depend on the Kubernetes setup.
+Automation may appoint the wrong leader - losing writes -
+or cause split-brain situations.
 
-Note that their use of the word _Node_ refers to a Pod and its volume in the Kubernetes world. So to change the `safe_to_bootstrap` flag, start a new pod with the correct volume attached.
+We can however support detection in the init script.
 
-### Summary (If link becomes broken)
+It's normal operations to scale down to two instances
+- actually one instance, but nodes should be considered ephemeral so don't do that -
```

Comment:
Which means that we'd handle losing all but 1 pod (due to for example zone or node failures) without getting nervous? And without manual intervention?

Reply:
Yes. I had a feeling that sentence needed clarification.